custom_components/tuya_local/devices/fukumaru_af01w_pet_feeder.yaml

```diff
@@ -0,0 +1,93 @@
+name: FUKUMARU pet feeder
+products:
+  - id: jttvwskiffndnb1q
+    manufacturer: FUKUMARU
+    model: FUKUMARU-W
+    model_id: AF01-W
+entities:
+  - entity: button
+    name: Quick feed
+    icon: "mdi:food-drumstick"
+    dps:
+      - id: 2
+        type: boolean
+        name: button
+        mapping:
+          - dps_val: true
+            value: true
+  - entity: number
+    name: Manual feed
+    icon: "mdi:food-drumstick"
+    dps:
+      - id: 3
+        type: integer
+        name: value
+        unit: portions
+        range:
+          min: 1
+          max: 6
+  - entity: sensor
+    class: enum
+    translation_key: status
+    category: diagnostic
+    dps:
+      - id: 4
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: standby
+            value: standby
+          - dps_val: feeding
+            value: feeding
+          - dps_val: done
+            value: feeding_complete
+  - entity: sensor
+    name: Feed report
+    icon: "mdi:food-drumstick"
+    category: diagnostic
+    dps:
+      - id: 5
+        type: integer
+        name: sensor
+        unit: portions
+  - entity: sensor
+    name: Battery
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 10
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: number
+    name: Voice playback count
```

Comment:
"count" implies a sensor, but this is a number entity, settable by the user. What is the purpose of this?